daemon/src/maker_inc_connections.rs

```diff
@@ -157,9 +157,16 @@ impl Actor {
     async fn handle_broadcast_order(&mut self, msg: BroadcastOrder) -> Result<()> {
         let order = msg.0;
         for (taker_id, conn) in self.write_connections.clone() {
-            tracing::trace!(%taker_id, "sending new order for broadcast to connection: {:?}", order);
-            conn.do_send_async(wire::MakerToTaker::CurrentOrder(order.clone()))
-                .await?;
+            if conn
+                .do_send_async(wire::MakerToTaker::CurrentOrder(order.clone()))
+                .await
+                .is_err()
```

Comment:
I think it would be good to encapsulate this into a function that we use everytime we send something.